feature-libs/pickup-in-store/components/presentational/store/store.component.html

```diff
@@ -42,19 +42,23 @@
     <div>
       <cx-icon
         class="cx-store-stock-icon"
-        *ngIf="!!storeDetails?.stockInfo?.stockLevel"
+        *ngIf="isInStock()"
```

Comment:
This should be a variable instead of function call.

Reply:
Done